protocol/src/reqres.rs

```diff
@@ -25,11 +25,16 @@ use libp2p::request_response::RequestResponseCodec;
 use std::io;
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct Req(pub Vec<u8>);
+pub enum Req {
+    Ping (),
+    Get (Vec<u8>),
+}
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct Resp(pub Vec<u8>);
-
+pub enum Resp {
+    Pong (),
+    Data (Vec<u8>),
+}
 impl ProtocolName for ReqRespProtocol {
```

Comment:
Let's have a representative name for the protocol, like `RobonomicsProtocol`. And `RobonomicsCodec` for codec.